diagnostic.js

```diff
@@ -29,29 +29,82 @@ mongoose.Promise = global.Promise;
 mongoose.connect('mongodb://localhost/mongoose-crud');
 
 /// ADD YOUR CODE BELOW
+const db = mongoose.connection;
+const Art = require('./art.js');
```

Comment:
`Movement` was the naming I was looking for, but this is fine as long as you're consistent.